CHANGELOG.md

```diff
@@ -0,0 +1,57 @@
+# Changelog
```

Comment:
Thanks - this changelog will need manual maintenance right? At the moment, every release, we generate the changelog with the release notes.

![Screenshot 2020-12-18 at 17 49 02](https://user-images.githubusercontent.com/1465130/102644890-7d98e900-4159-11eb-9307-240f23fd8a87.png)

Reply:
https://github.com/TykTechnologies/tyk-operator/releases/tag/v0.4.0 as an example.

Is there a benefit in doing it one way over the other?

Reply:
If you're happy with the subject lines from the commits/PRs forming the body of the changelog like this, then you don't need to re-invent this particular wheel 🙂 
The [0.4.0](https://github.com/TykTechnologies/tyk-operator/releases/tag/v0.4.0) and [0.4.1](https://github.com/TykTechnologies/tyk-operator/releases/tag/v0.4.1) releases do look like they have plenty of meat on the bone as-is 👍

Reply:
9bbdba5 resolves